src/main/kotlin/io/curity/identityserver/plugins/authenticator/authentication/CallbackRequestHandler.kt

```diff
@@ -95,6 +95,26 @@ class CallbackRequestHandler(
         return Optional.ofNullable(AuthenticationResult(attributes))
     }
 
+    private fun fetchUserInfoClaims(providerAccessToken:String): JwtClaims{
+        val userInfoResponse = _config.getHttpClient()
+            .request(_providerConfiguration.userInfoEndpoint)
+            .header("Authorization", "Bearer $providerAccessToken")
+            .post()
+            .response()
+
+        return JwtClaims.parse(userInfoResponse.body(HttpResponse.asString()))
```

Comment:
This will only work if the userinfo response is JSON. Thats probably enough for this case, but we should note it it in the README

Reply:
Fixed.